docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,601 @@
 ---
-title: Teleport Trusted Clusters
-description: How to configure access and trust between two Teleport Clusters.
-h1: Trusted Clusters
+title: Configure Trusted Clusters
+description: Explains how you can configure a trust relationship and manage access between two Teleport clusters.
+h1: Configure trusted clusters
 ---
 
-Teleport can partition compute infrastructure into multiple clusters. A cluster
-is a group of resources connected to a Teleport Auth Server. Each cluster
-manages a separate certificate authority (CA) for all users and resources.
+As you learned in [Core Concepts](../../core-concepts.mdx), a Teleport cluster
+consists of the Teleport Auth Service, the Teleport Proxy Service, and the 
+Teleport services that manage access to resources in your infrastructure.
+With Teleport, you can also partition your infrastructure into multiple connected 
+clusters to allow the users of one cluster—the **root cluster**—to connect to resources 
+in another cluster—the **leaf cluster**—while authenticated using a single 
+Teleport Auth Service. 
 
-Trusted Clusters allow the users of one cluster, the **root cluster**, to
-seamlessly connect to resources in another cluster, the **leaf cluster**, while
-remaining authenticated with only a single Auth Service. The leaf cluster can
-run behind a firewall without any ingress ports open.
+After you establish a trust relationship between the root and leaf clusters, the leaf cluster can
+run behind a firewall without any inbound ports open. The leaf cluster creates an outbound reverse 
+SSH tunnel to the root cluster and keeps the tunnel open.
+When a user tries to connect to any resource in the leaf cluster, the connection goes through the 
+reverse tunnel using the root cluster's proxy.
 
-Uses for Trusted Clusters include:
+## Who uses trusted clusters?
```

Comment:
As with the other PR, I'm wondering if we should lowercase "Trusted Cluster" throughout the docs since it's a Teleport product. If we do decide to lowercase it, we should do so in the Core Concepts page as well.

Reply:
Yes, I think we should normalize case for common words.

Reply:
In that case, would you mind lowercasing it in "Core Concepts" so we can keep the docs consistent?

Also, what if a common term is also a product name? E.g., if we were to lowercase "database service", it could create confusion around the Teleport Database Service vs. some other database service (e.g., a Postgres server).